crypto/kdf/sskdf.c

```diff
@@ -0,0 +1,496 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2019, Oracle and/or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Refer to https://csrc.nist.gov/publications/detail/sp/800-56c/rev-1/final
```

Comment:
Hmmm, looking at this, I'm wondering if commenting that's not the copyright+license boilerplate shouldn't be a *separate* comment...

Reply:
Good point, I agree that it should be a separate comment.